app/screens/Main/index.js

```diff
@@ -9,7 +9,7 @@ export default class Main extends Component {
       
       this.state = {
           textInputValue: '',
-          warning: false
+          warning: false,
```

Comment:
Seriously???

Reply:
Yeah, it's a good practice to use dangling commas :) https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
Convinced!